tools/test/heuristics/test_heuristics.py

```diff
@@ -6,10 +6,10 @@
 from typing import Any, Dict, Set
 from unittest import mock
 
-REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent.parent
+REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent.parent.parent
 try:
     # using tools/ to optimize test run.
-    sys.path.append(str(REPO_ROOT))
+    sys.path.insert(0, str(REPO_ROOT))
```

Comment:
Out of curiosity, why was this needed? And should this become our new, universal pattern?

Reply:
This was originally me trying to figure out why I couldn't run the file, but we do this in run_test.py to ensure that we pick up the tools folder in pytorch and not the tools folder in some other repo, which someone encountered as an error becore